sdk/storage/azure-storage-queue/README.md

```diff
@@ -132,9 +132,14 @@ https://myaccount.queue.core.windows.net/myqueue
 ```
 
 ### Handling Exceptions
+Suppose you have the `queueServiceClient` already
```

Comment:
instead of this sentence, why not add a function signature to the example which takes a QueueServiceClient?

Reply:
The code to build the client everywhere in readme. I will add a link to direct to the section of build client code snippet.